Alertinator.php

```diff
@@ -252,16 +249,25 @@ protected function sms(string $number, string $message): void {
    protected function call(string $number, string $message): void {
       $twiml = $this->getTwiML($message);
       $number = '+1' . $number;
-      $this->getTwilioCall()->create(
-       $number, $this->twilio['fromNumber'], ['Twiml' => $twiml]);
+      $this->getTwilioCall()->create($number, $this->twilio['fromNumber'], [
+         'Twiml' => $twiml,
+         'machineDetection' => 'Enable',
+      ]);
    }
 
    protected function getTwiML(string $message): VoiceResponse {
       $twiml = new VoiceResponse();
       $twiml->say($this->getVoicePrefix());
       $twiml->say($message);
-      $twiml->say(self::CONFERENCE_MESSAGE);
-      $twiml->dial()->conference(self::CONFERENCE_NAME);
+
+      // Connect the user to the conference call only if they press a
+      // digit on the phone. This seems more reliable than answering machine
+      // detection.
+      $gather = $twiml->gather([
+         'action' => 'https://handler.twilio.com/twiml/EHf60b496ecfab6bd73cc23f13e97c8f66',
+         'numDigits' => 1,
+      ]);
+      $gather->say('Press any digit to connect to conference');
```

Comment:
Is this going to `say` the whole alert message non-stop? Like "alert wharrgarll threshold one press any digit to connect"? You've been exercising this @sterlinghirsh so i'll defer to you, but is there a way to put in, say, a one second pause before the conference connect? Just a thought.

Reply:
It does say the whole thing, but it does have slight pauses on punctuation. I wouldn't mind putting in a 1s delay though.